website/docs/azure/quickstart.md

```diff
@@ -0,0 +1,187 @@
+
+### Create a New Virtual Machine
+
+You can create a new virtual machine in the same virtual network as your Azure Cosmos DB Garnet cache.
+- [Create a Linux VM](https://learn.microsoft.com/azure/virtual-machines/linux/quick-create-portal?tabs=ubuntu)
+- [Create a Windows VM](https://learn.microsoft.com/azure/virtual-machines/windows/quick-create-portal)
+
+### Use Existing Infrastructure
+
+You can use an existing VM or host machine in the same virtual network as your Azure Cosmos DB Garnet Cache. If your VM is in a different virtual network, [set up virtual network peering](https://learn.microsoft.com/azure/virtual-network/tutorial-connect-virtual-networks?tabs=portal). Ensure that the IP address spaces of the two virtual networks do not overlap. Once peering is established successfully, the client application in one virtual network can access the cache endpoints on the other network using their local IP addresses.
+
+## Step 4: Connect and Test
+
+Now you're ready to connect to your Azure Cosmos DB Garnet Cache!
+
+1. Find your user Object ID. Save this for a future step.
+
+```bash
+# Get your own Object ID
+az login
+az ad signed-in-user show --query id -o tsv
+```
+
+2. Get an access token. Save this for a future step.
+
+```bash
+az account get-access-token --scope https://cosmos.azure.com/.default --query accessToken -o tsv
+```
+
+3. Find the IP address of your cache nodes. Redis clients can connect to one of the node IP addresses and get the list of all replicas and ports automatically.
+    1. Sign in to the [Azure portal](https://aka.ms/garnet-portal). The Azure Cosmos DB Garnet Cache is in an expanded Private Preview and you must access the Azure portal through this link to see your caches.
+    2. Navigate to the **Settings > Cluster Explorer** page of your Azure Cosmos DB Garnet Cache resource.
+    3. Find the IP addresses of all nodes in your cluster. Save them for a future step.
+
+    ![Find Azure Cosmos DB Garnet Cache node IP addresses](../../static/img/azure/cluster-ip-addresses.png)
+
+4. Connect to a Redis client. You can use any Redis client of your choice to connect to the cluster.  Connecting with the [Redis CLI](https://redis.io/docs/latest/develop/tools/cli/) is optional, and it provides a quick way to test data access for your cache.
+    1. [Install the Redis CLI](https://redis.io/docs/latest/develop/tools/cli/#install-redis-cli) on your machine. If you're using a Linux machine, run the following command.
+
+    ```bash
+    sudo apt-get install redis-tools
+    ```
+
+    2. Connect to your cluster using the Redis CLI.
+
+    ```bash
+    # Replace with your actual values
+    export USER_OBJECT_ID="your-object-id-from-step-1"
+    export ACCESS_TOKEN="your-access-token-from-step-2"
```

Comment:
Do we want to mention here or at Step 2 that access tokens have an expiry window and regenerate the token if it has expired.